CONTRIBUTING.md

```diff
@@ -1,61 +1,520 @@
 # Contributing Guidelines
 
-Thank you for your interest in contributing to our project. Whether it's a bug report, new feature, correction, or additional
-documentation, we greatly value feedback and contributions from our community.
+Thank you for your interest in contributing to the Android Amplify
+framework. Whether it's a bug report, new feature, correction, or
+additional documentation, the project maintainers at AWS greatly value
+your feedback and contributions.
 
-Please read through this document before submitting any issues or pull requests to ensure we have all the necessary
-information to effectively respond to your bug report or contribution.
+Please read through this document before submitting any issues or pull
+requests. Doing so will help to ensure that the project maintainers have
+all information neessary to effectively respond to your bug report or
+contribution.
 
+- [Contributing Guidelines](#contributing-guidelines)
+  * [Amplify History and Ethos](#amplify-history-and-ethos)
+    + [Amplify Design](#amplify-design)
+  * [Getting Started](#getting-started)
+  * [Tools](#tools)
+  * [Workflows](#workflows)
+    + [Adding code to support a new feature](#adding-code-to-support-a-new-feature)
+  * [Reporting Bugs/Feature Requests](#reporting-bugs-feature-requests)
+  * [Contributing via Pull Requests](#contributing-via-pull-requests)
 
-## Reporting Bugs/Feature Requests
 
-We welcome you to use the GitHub issue tracker to report bugs or suggest features.
+## Amplify History and Ethos
+AWS Amplify aims to simplify the experience of developing cloud
+integrations on Android. Amplify codifies best practices through
+programmatic interfaces that reduce the effort needed to interact with
+cloud resources.
+
+First and foremost, Amplify exposes to you _what_ things do, and then
+_how_ best to do them. The "_what_" is a functional use case, and
+"_how_" is an opinionated implementation that you can override with
+"escape hatches." This will allow you to have higher velocity and build
+better applications by focusing less on implementation choices.
+
+Secondly, Amplify is a manifestation of
+[The Rule of Least Power](https://en.wikipedia.org/wiki/Rule_of_least_power)
+when developing against AWS. This means it encourages architectural and
+programmatic best practices that enable you to start quickly.  This
+shows by encouraging certain services (API Gateway usage vs. direct
+DynamoDB interaction) or certain connection patterns (Circuit breaker,
+retry counts and throttle up/down).
+
+__Opinionated implementations__: There are many ways to interface with AWS
+Services. Certain service interactions are favored over others. For instance, if
```

Comment:
Nit: Why is Service a proper noun in this document?

Reply:
Not sure; I agree it should not be. This was copy-pasted. For this PR, I've resolved this by removing the containing section from the `CONTRIBUTING.md`.